addrmgr/addrmanager_test.go

```diff
@@ -498,3 +498,26 @@ func TestNetAddressKey(t *testing.T) {
 	}
 
 }
+
+func TestCorruptPeersFile(t *testing.T) {
+	dir, err := ioutil.TempDir("", "testaddressbyip")
```

Comment:
Shouldn't the prefix be changed here to be unique for the test?

Reply:
Good catch, changed accordingly!